nym-node-status-api/src/db/mod.rs

```diff
@@ -0,0 +1,32 @@
+use crate::read_env_var;
+use anyhow::{anyhow, Result};
+use sqlx::{migrate::Migrator, SqlitePool};
+pub(crate) const DATABASE_URL_ENV_VAR: &str = "DATABASE_URL";
+static MIGRATOR: Migrator = sqlx::migrate!("./migrations");
+
+pub(crate) mod helpers;
+pub(crate) mod models;
+pub(crate) mod queries;
+
+pub(crate) type DbPool = SqlitePool;
+
+pub(crate) struct Storage {
+    pool: DbPool,
+}
+
+impl Storage {
+    pub async fn init() -> Result<Self> {
+        let connection_url = read_env_var(DATABASE_URL_ENV_VAR)?;
+
+        let pool = sqlx::SqlitePool::connect(&connection_url)
```

Comment:
> What does it connect to if you just run without setting DATABASE_URL_ENV_VAR?

nothing, it fails during start up because it needs DB path

> I would add some logging here so that when you run it, it will log where it's storing stuff. I ran it locally and it wasn't showing anything.

it [does log](https://github.com/nymtech/nym/blob/97642507e21dc9d96ed5221247ed72d9991b3777/nym-node-status-api/src/main.rs#L35) env var after successfully reading it

Reply:
> Also I suggest using disable_statement_logging() here to not flood the log. That way you can remove that logging directive for sqlx, so that we don't hide other relevant logging for it.